app/containers/MessageBox/index.js

```diff
@@ -745,14 +842,61 @@ class MessageBox extends Component {
 				<FlatList
 					style={styles.mentionList}
 					data={mentions}
-					renderItem={({ item }) => this.renderMentionItem(item)}
-					keyExtractor={item => item._id || item.username || item}
+					renderItem={this.renderMentionItem}
+					keyExtractor={item => item._id || item.username || item.command || item}
 					keyboardShouldPersistTaps='always'
 				/>
 			</View>
 		);
 	};
 
+	renderCommandPreviewItem = ({ item }) => {
+		if (item.type === 'image') {
+			return (
+				<TouchableOpacity
```

Comment:
you have this piece of code duplicated in the same function:
 ```
<TouchableOpacity
					style={styles.commandPreview}
					onPress={() => this.onPressCommandPreview(item)}
					testID={`command-preview-item${ item.id }`}
				>
```
Do the following: 
```
<TouchableOpacity>{item.type === 'image' ? <FastImage> : <CustomIcon>}</TouchableOpacity>
```